docs-parts/computation/06-distributed-computing_jobs_by_key.rst

```diff
@@ -0,0 +1,21 @@
+
+This can be done by using `dj.hash.key_hash` to convert the key as follows:
+
+.. code-block:: python
+
+    In [4]: schema.jobs & {'key_hash' : dj.hash.key_hash({'id': 2})}                
```

Comment:
Makes sense. Based on the discussion, let's update this doc to:
```suggestion
    In [4]: schema.jobs & {'key_hash' : dj.key_hash({'id': 2})}                
```

Reply:
thanks, marking feedback as resolved by 25ae98f